frontend/src/scenes/batch_exports/components.tsx

```diff
@@ -39,6 +39,7 @@ export const colorForStatus = (
         case 'TimedOut':
             return 'warning'
         case 'Failed':
+        case 'FailedRetryable':
```

Comment:
@tiina303 @tomasfarias As noted in the PR description, for now this PR just treats these two error types the same on the frontend. So there should be no visible change to users. I figured a future PR could do UI work if/when we desire, I didn't want to bundle it in here, especially since I'm rather useless there.

Reply:
`FailedRetryable` should have a link to contact support as that we should strive to never raise retryable errors to users.